src/tsf/TfDispAttr.cpp

```diff
@@ -64,7 +64,7 @@ CicDisplayAttributeMgr::~CicDisplayAttributeMgr()
             }
 
             wil::com_ptr<ITfReadOnlyProperty> pProp;
-            if (SUCCEEDED(hr = pic->TrackProperties(ppguidProp.get(), ulNumProp, 0, NULL, &pProp)))
+            if (SUCCEEDED(hr = pic->TrackProperties(ppguidProp.get(), ulNumProp, nullptr, NULL, &pProp)))

```

Comment:
I don't know if this is the reason why the builts fail but seems to be still incorrect anyway.